include/nccl_ofi_rdma.h

```diff
@@ -435,6 +435,8 @@ typedef struct nccl_net_ofi_rdma_send_comm {
 	 * and its base struct. */
 	nccl_net_ofi_send_comm_t base;
 
+	bool comm_active;
```

Comment:
why here near the top of the struct, instead of elsewhere?

Reply:
more importantly, a bool is going to be no more than 32 bits on modern compilers, and so you've forced 32 bits of padding between `comm_active` and `num_inflight_reqs` for no real win.

Reply:
Moving to end of struct (right before the flexible array member at the end of the comm structs).